Documentation/ceph-block.md

```diff
@@ -142,6 +142,150 @@ kubectl delete -n rook-ceph cephblockpools.ceph.rook.io replicapool
 kubectl delete storageclass rook-ceph-block
 ```
 
+## RBD Snapshots
+
+### Snapshot Prerequisites
+
+1. Requires kubernetes v1.17+ which supports snapshot beta.
```

Comment:
If someone still wants to use the 2.x driver, they can still use alpha snapshots, right? In that case, what if we add a link back to the rook 1.3 documentation to indicate they can still use it if they don't have 1.17.

Reply:
No, we should avoid that one, it's of no use as the Alpha snapshots are not backward compatible, Even once the CSI driver is upgraded they cannot delete the snapshots.

Reply:
While hopefully most users are on 1.17 or newer already, some users stay on older versions for a long time. For example, if someone is still on 1.16 and they really want to use the alpha snapshots, what would they do? Is their only option to use rook v1.3? What I'm suggesting is that if the csi min version is still 2.x, they could still choose to use the alpha snapshots, and they would need to go through the upgrade steps to delete them before enabling the 3.0 driver. But if there is some issue supporting this version compatibility, we can remove it, I just want to be clear if there is a challenge to support it.